server/lib/manifest.py

```diff
@@ -319,6 +320,13 @@ def _parse_dataSet(self, dataSet=None, relpath=''):
                 objects from external_objects
 
         """
+        if dataSet is None and self.version is None and self.is_export is True:
```

Comment:
can you explain why this is needed? Also logic in that if statement is "iffy". Cause if `self.version is None` the next line will die horrible death...

Reply:
Fixed that since there will [always](https://github.com/whole-tale/girder_wholetale/blob/7c740b66a5e7c16dabc5a328dcce5b35321d3ce8/server/lib/manifest.py#L50) be something in `self.version`.

I check to see if the manifest is being used during an export event so that we know manifest.json is present on disk. If its not, then we revert to the actual Tale record for finding out dataSet.